data/hai/hai reveal 4 middle.txt

```diff
@@ -1456,32 +1458,410 @@ mission "Hai Reveal [A11] Sound the Alarm"
+				goto reading
+				to display
+					has "Hai Reveal [A11] Sound the Alarm: human book"
+
+			label unfettered
+			action
+				log `A Hai intelligence report puts the cost to Hai of the Unfettered raids at 3.5 billion credits a day, while the cost is expected to double in "full Hai unification." Unification costs include 5 billion credits a day of "military aid."`
+				log `A Hai intelligence report on "Unfettered military activities" included several fully-redacted sections, including "Multi-Decadal Timeline Expected before Unfettered Progress," and "Theories on the Sentient Evolution of Nocturnal Raptors."`
+				set "Hai Reveal [A11] Sound the Alarm: unfettered book"
+				"Hai Reveal [A11] Sound the Alarm: time" ++
+			`	This report begins with an economic analysis of the cost of raids at Wah Ki, which it places at a staggering 3.5 billion credits per day. This is 1.5 billion impact on trade; 1.5 billion for military fleet repairs, replacements, ammunition, and infrastructure to support it; and an estimated 500 million in humanitarian aid to the Unfettered, to prevent the problem from worsening. It compares that to "full Hai unification" for which the cost is 7 billion credits per day: 3 billion in humanitarian aid to the Unfettered, 5 billion in military aid, but 1 billion per day recouped through improved trade.`
+			`	The remaining 80% of the report is redacted in its entirety, save for a few section headings:`
+			`	-	Multi-Decadal Timeline Expected before Unfettered Progress`
+			`	-	Choot'k: Insane Risk-Taker or Brilliant Strategist?`
+			`	-	Reports from Hai Agents on Unfettered Raiding Vessels`
+			`	-	Unsustainable Rate of Losses Spying on Unfettered Leadership`
+			`	-	Theories on the Sentient Evolution of Nocturnal Raptors`
+			`	Musing on the titles, you recall that "raptors" are birds of prey. Why would an intelligence report on the Unfettered discuss nocturnal birds of prey?`
+				goto reading
+
+			label acorns
+			action
+				log `A Hai intelligence report said Unfettered planets have been ruined to the point where they cannot only produce enough food for bare sustenance. Birds, predators, and large species were almost wiped out, destroying the ecosystem. It mentioned three species of oak that could grow, but none are ideal. As time goes on Unfettered will increasingly rely on external food sources.`
+				set "Hai Reveal [A11] Sound the Alarm: acorns book"
+				"Hai Reveal [A11] Sound the Alarm: time" ++
+			`	This report describes the catastrophic environmental collapse of the Unfettered worlds through over-farming, lack of pollution controls, and over-mining. Nearly all large species, including predators and nearly all birds, have been wiped out, ruining the ecosystems. Presently, the Unfettered can produce slightly more food needed for bare sustenance on their own, and are reliant on donations or raids for the rest.`
```

Comment:
The Unfettered situation is bad but not *that bad*
Most food prod happens on one planet (Warfeed), and in normal times it is barely ok.
The issue is there often are events causing the situation to degenerate (a blight, a natural disaster...) So they dont have any stockpiles and most dont eat as much as they would like in normal times but its ok
I portrayed the food crisies as happening every 500 days and lasting about 150

They raid to have a stockpile to survive during those times

Reply:
I have updated it. How does it look now?